scripts/validate_unwanted_patterns.py

```diff
@@ -83,23 +84,48 @@ def bare_pytest_raises(file_obj: IO[str]) -> Iterable[Tuple[int, str]]:
     -----
     GH #23922
     """
-    tokens: List = list(tokenize.generate_tokens(file_obj.readline))
 
-    for counter, current_token in enumerate(tokens, start=1):
-        if not (current_token.type == token.NAME and current_token.string == "raises"):
-            continue
-        for next_token in tokens[counter:]:
-            if next_token.type == token.NAME and next_token.string == "match":
+    def get_fqdn(node: ast.AST) -> Optional[str]:
```

Comment:
Instead of this I think you can just call `node.func.value.id` to get pytest and `node.func.attr` for the raises portion